### DISCUSSION TITLE
Руководство ПЛИ, п.3.7 Сведения об источнике оплаты <participant [@typeCode="IND"]>

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#275] [Лабораторная РГ]
Элемент participant [@typeCode="IND"] используется для указания информации об источнике оплаты и может содержать информацию, значительно отличающуюся по наполнению. По этой причине большинство элементов указаны с возможностью использования атрибута @nullFlavor или могут быть полностью убраны из документа, что является не всегда корректным. В Таблица 9 указаны поля, которые обязательны к заполнению при указании различных типов источника оплат в соответствии с актуальной на момент разработки руководства версией 5.2 справочника "Источники оплаты медицинской помощи" (OID: 1.2.643.5.1.13.13.11.1039).

**Comment by eugenekogan-spb** [comment]
<img width="766" alt="image" src="https://github.com/user-attachments/assets/0747faad-2aeb-4104-b521-d24de14181d0">
<img width="763" alt="image" src="https://github.com/user-attachments/assets/65c127f1-b504-4209-a86b-8e3b57dc3f37">
<img width="765" alt="image" src="https://github.com/user-attachments/assets/aa5c89a3-3e1e-4788-a2c3-1a42534d37c6">

**Reply by eugenekogan-spb** [reply]
Пример аналогичный приведенному в руководстве ПЛИ

```
//Пример аналогичный приведенному в руководстве ПЛИ


CodeSystem: Core_Cs_Nsi_Identity_Document
Id: core-cs-nsi-identity-document
* #1 "Паспорт РФ"

CodeSystem: Core_Cs_Nsi_Sources_Of_Payment
Id: core-cs-nsi-sources-of-payment
* #1 "Средства ОМС"

CodeSystem: Core_Cs_Nsi_Coverage_Document
Id: core-cs-nsi-coverage-document
* #1 "Полис ОМС"

CodeSystem: Core_Cs_Nsi_Coverage_Document_OMS
Id: core-cs-nsi-coverage-dociment-oms
* #1 "Полис ОМС старого образца"
* #2 "Полис ОМС единый, бессрочный"
* #3 "Полис ОМС единый, со сроком"
* #4 "Временное свидетельство ОМС"


Instance: lab-coverage-instance-oms
InstanceOf: Coverage
Title: "Lab Instance Coverage OMS"
Description: "Пример источника оплаты, идентичный руководству СЭМД ПЛИ"
Usage: #example

* status = #active

* type = Core_Cs_Nsi_Sources_Of_Payment#1

//* payor = Reference(lab-organization-instance-RESO-med-altay)
* payor
  * identifier[+]
    * system = "http://fhir.ru/core/systems/ffoms_smo"
    * value = "22001"

* beneficiary = Reference(patient-id-3)

* identifier[+]
  * system = "http://fhir.ru/core/systems/OMS_main_unlimited"
  * value = "9876543211234567"
  * type = Core_Cs_Nsi_Coverage_Document_OMS#2
 
* period
  * start = "2017-10-25"

//Экземпляр организации приведен только для визуального соответствия примеру в СЭМД.
//В пакете он не нужен, потому что конвертер должен взять актуальную версию из реестра ТФОМС в момент конвертации.

Instance: lab-organization-instance-RESO-med-altay
InstanceOf: Organization
Usage: #inline


//<!-- R [1..1] Наименование медицинской организации или ФИО ИП -->
* name = "АЛТАЙСКИЙ ФИЛИАЛ ОБЩЕСТВА С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ СТРАХОВАЯ МЕДИЦИНСКАЯ
КОМПАНИЯ РЕСО-МЕД"

* identifier[+]
  * system = "http://fhir.ru/core/systems/ffoms_smo"
  * value = "22001"


//<!-- [0..*] Контакты медицинской организации или ИП -->
* telecom[0]
  * system = #phone
  * value = "+7 385 255 67 67"
  * use = #mobile

//<!-- R [1..1] Адрес медицинской организации или ИП -->
* address
  //<!-- R [1..1] Адрес (текстом) -->
  * text = "Алтайский край, город Барнаул, улица Мало-Олонская, дом 17"

  //<!-- R [1..1] Код субъекта РФ -->
  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #2
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "Алтайский край"
        
  //<!-- [1..1] Почтовый индекс -->
  * postalCode = "656043"
  
  
  ////<!-- [1..1] Идентификатор адреса по ФИАС -->
  //* extension[fias]
  * extension[+]
    * url = "fias"
    ////<!-- R [1..1] Глобальный уникальный идентификатор адресного объекта -->
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "358ecfff-a458-40c5-be88-3612366736d2"
        * system = "urn:hl7-ru:fias:aoguid"
    ////<!-- [1..1] Глобальный уникальный идентификатор дома -->
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "07cdd2de-ca6b-420f-a5c0-5d99268ff28c"
        * system = "urn:hl7-ru:fias:houseguid"


Instance: patient-id-3
InstanceOf: Patient
Usage: #inline

* active = true

* name
  * text = "Иванов Иван"
  
* managingOrganization = Reference(organization-id-4)
  
//3.2.1а Уникальный идентификатор пациента в МИС
//<!-- R [1..1] Уникальный идентификатор пациента в МИС -->
//<id root="1.2.643.5.1.13.13.12.2.22.1576.100.1.1.10" extension="3421"/>
//identifier[misPatientId] (OID_медицинской_организации.100.НомерМИС.НомерЭкзМИС.10)
* identifier[0]
  * value = "3421"
  * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.10" 
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#PRN

//3.2.1б СНИЛС
//* identifier[SNILS] 
* identifier[+]
  * value = "152-873-614 14"
  * system = "http://fhir.ru/core/systems/snils"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SB

//3.2.1.в Документ, удостоверяющий личность  
//* identifier[passportRF]
* identifier[+]
  * value = "7412 809982"
  * system = "http://fhir.ru/core/systems/passport-RF"
  * type = Core_Cs_Nsi_Identity_Document#1  
  
//3.2.1г Полис ОМС  
* identifier[+]
  * value = "123456 1234567890"
  * system = "http://fhir.ru/core/systems/OMS"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SN
  
//3.2.1.д Адрес
* address 

  * text = "Россия, г. Санкт-Петербург, Софийская ул., д.32-1-987"
  
  // * extension[addressType]
  * extension[0]
    * url = "addressType"
    * valueCoding
      * code = #1
      * system = "1.2.643.5.1.13.13.11.1504"
      * version = "1.4"         
      * display = "Адрес по месту жительства (постоянной регистрации)"

  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #77
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "г. Москва"

  * postalCode = "192236"

  //* extension[fias]
  * extension[+]
    * url = "fias"
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:aoguid"
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:houseguid"

//3.2.1.е контакты
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile
  
//3.2.1.ж Личные данные

* name
  * family = "Габсбург-Лотарингская"
  * given[0] = "Мария-Антуанетта"
  * given[1] = "Иоганновна"
  * use = #official

* gender = #female

* birthDate = "1955-01-10"

Instance: organization-id-4
InstanceOf: Organization
Usage: #inline

//<!-- R [1..1] Наименование медицинской организации или ФИО ИП -->
* name = "ОБЩЕСТВО С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ \"ХЕЛИКС АЛТАЙ\""

//<!-- [0..*] Контакты медицинской организации или ИП -->
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile

//<!-- R [1..1] Адрес медицинской организации или ИП -->
* address
  //<!-- R [1..1] Адрес (текстом) -->
  * text = "190005, Россия, г. Санкт-Петербург, наб. р. Фонтанки, д. 132"

  //<!-- R [1..1] Код субъекта РФ -->
  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #2
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "Алтайский край"
        
  //<!-- [1..1] Почтовый индекс -->
  * postalCode = "656067"
  
  
  ////<!-- [1..1] Идентификатор адреса по ФИАС -->
  //* extension[fias]
  * extension[+]
    * url = "fias"
    ////<!-- R [1..1] Глобальный уникальный идентификатор адресного объекта -->
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "796de636-5fd9-4f3d-97ad-c865c60a024d"
        * system = "urn:hl7-ru:fias:aoguid"
    ////<!-- [1..1] Глобальный уникальный идентификатор дома -->
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "19229ca9-d457-4f07-ab86-8d64c868502a"
        * system = "urn:hl7-ru:fias:houseguid"

//<!-- [0..1] Код ОГРНИП -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОГРНИП"
  * value = "4719008550"
  
//<!-- [0..1] Код ОКПО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКПО"
  * value = "03703245"
  
//<!-- [0..1] Код ОКАТО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКАТО"
  * value = "01401364000"  

//<!-- R [1..1] Уникальный идентификатор медицинской организации и (при наличии) уникальный идентификатор структурного подразделения -->
//* identifier[FRMO]
* identifier[+]
  * system = "http://fhir.ru/core/systems/frmo"
  * value = "1.2.643.5.1.13.13.12.2.22.27143"

//<!-- [0..1] Сведения о лицензии на осуществление медицинской деятельности -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/medlicense"
  * value = "Л041-01151-22/00376231"
  * assigner
    * display = "Министерство здравоохранения Алтайского края. Дата регистрации: 29.05.2020"


```

**Comment by eugenekogan-spb** [comment]
1.Средства ОМС (1 - полис ОМС старого образца, 2 - полис ОМС единого образца, бессрочный, 3 - полис ОМС единого образца, со сроком действия, 4 - временное свидетельство)
3.Средства ДМС
4.Пациент
5.Иное физ лицо
6.Иное юр лицо

<img width="777" alt="image" src="https://github.com/user-attachments/assets/368f89fa-91ff-4891-984f-8f80420d1204">

**Reply by eugenekogan-spb** [reply]
Неожиданно: при указании источника финансирования - средства пациента, иного юридического или физического лица, в СЭМД не требуется указание ИНН, хотя он необходим для взаиморасчетов. Может быть, мы напрасно включили в RuCore для этих ситуаций обязательность ИНН?

**Reply by eugenekogan-spb** [reply]
На встрече 15.10.24 обсудили, что для решения этой коллизии стоит в RuCore в профилях Patient_Payer, Organization_Payer, Related_Person_Payer поменять требование обязательности ИНН на MustSupport.

Вопрос: можно ли сделать MustSupport на позицию слайсинга?
@Vrodvi

**Comment by eugenekogan-spb** [comment]
3.7.1. Источник оплаты
У1-18:Элемент ClinicalDocument МОЖЕТ иметь один [0..1] элемент participant (атрибут @typeCode должен иметь значение "IND"), содержащий сведения об источнике оплаты, который:
1. ОБЯЗАН иметь один [1..1] элемент associatedEntity (атрибут @classCode должен иметь значение "GUAR"), который:
 по справочнику ‟Источники оплаты медицинской помощи‟ (OID 1.2.643.5.1.13.13.11.1039): 1 - средства обязательного медицинского страхования, 3 - средства добровольного медицинского страхования, 4 - средства пациента, 5 - средства третьих физических лиц, 6 - средства третьих юридических лиц, 8 - средства федерального бюджета, 9 - средства регионального бюджета, 10 - средства обязательного социального страхования, 11 - средства бюджета медицинской организации, 12 - Средства федерального и регионального бюджета.
5 по справочнику ‟Виды полиса обязательного медицинского страхования‟ (OID 1.2.643.5.1.13.13.11.1035): 1 - полис ОМС старого образца, 2 - полис ОМС единого образца, бессрочный, 3 - полис ОМС единого образца, со сроком действия, 4 - временное свидетельство.

1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1039", атрибут @codeSystemName должен иметь значение "Источники оплаты медицинской помощи", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий тип источника оплаты;
2. ДОЛЖЕН иметь один [1..1] элемент identity:DocInfo, содержащий сведения о документе-основании. ЕСЛИ атрибут ClinicalDocument/ participant[@typeCode="IND"]/associatedEntity[@classCode="GUAR"]/code/@c ode имеет значение "8", "9", "10", "11" или "12", ТО атрибут identity:DocInfo/@nullFlavor должен иметь значение "NA" или "NAV". Данный элемент:

**Reply by eugenekogan-spb** [reply]
```
* type = Core_Cs_Nsi_Sources_Of_Payment#1
```

**Reply by eugenekogan-spb** [reply]
Дополнительно: этот источник оплаты должен к чему-то относиться, не к документу. Видимо, к ServiceRequest

```
InstanceOf: ServiceRequest
* insurance = Reference(coverage-oms)
```

**Comment by eugenekogan-spb** [comment]
3.7.2. Сведения о документе

1.ОБЯЗАН иметь один [1..1] элемент identity:IdentityDocType (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.724", атрибут @codeSystemName должен иметь значение "Типы документов оснований", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий тип документа- основания. Возможны следующие альтернативы:
1. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/code/@code имеет значение "1", ТО атрибут identity:IdentityDocType/@code должен иметь значение "1", атрибут identity:IdentityDocType/@displayName должен иметь значение "Полис ОМС";
2. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/code/@code имеет значение "3", ТО атрибут identity:IdentityDocType/@code должен иметь значение "2", атрибут identity:IdentityDocType/@displayName должен иметь значение "Полис ДМС";
3. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/code/@code имеет значение "4", "5" или "6", ТО атрибут identity:IdentityDocType/@code должен иметь значение "3", атрибут identity:IdentityDocType/@displayName должен иметь значение "Договор на оказание платных медицинских услуг";
2. ДОЛЖЕН иметь один [1..1] элемент identity:InsurancePolicyType (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1035", атрибут @codeSystemName должен иметь значение "Виды полиса обязательного медицинского страхования", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий вид полиса ОМС. Возможны следующие альтернативы:
1. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:IdentityDo cType/@code имеет значение "1", ТО:
1.ЕСЛИ оплата происходит по ОМС, но информация о полисе отсутствовала на момент создания документа, ТО атрибут identity:InsurancePolicyType/@nullFlavor должен иметь значение "NAV";
2. ЕСЛИ иначе, ТО атрибут identity:InsurancePolicyType/@code должен иметь значение в соответствии со справочником "1.2.643.5.1.13.13.11.1035";
2. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:IdentityDo cType/@code имеет значение "2" или "3", ТО атрибут identity:InsurancePolicyType/@nullFlavor должен иметь значение "NA";
3. ДОЛЖЕН иметь один [1..1] элемент identity:Series, содержащий серию документа-основания, который должен иметь не пустое наполнение. Возможны следующие альтернативы:
Руководство по реализации СЭМД "Протокол лабораторного исследования" в соответствии с третьим уровнем требований архитектуры клинических документов версии 2 (CDA 2.0 HL7 v.3)
 Страница 58
1. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@code имеет значение "1", ТО элемент identity:Series должен иметь не пустое наполнение;
2. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@code имеет значение "2", "3" или "4", ТО атрибут identity:Series/@nullFlavor должен иметь значение "NA";
3. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@nullFlavor имеет значение "NAV", ТО атрибут identity:Series/@nullFlavor должен иметь значение "NAV";
4. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@nullFlavor имеет значение "NA", ТО:
1.ЕСЛИ имеется серия у документа-основания, ТО элемент identity:Series должен иметь не пустое наполнение;
2. ЕСЛИ иначе, ТО атрибут identity:Series/@nullFlavor должен быть представлен с не пустым значением;
4. ДОЛЖЕН иметь один [1..1] элемент identity:Number, содержащий номер документа-основания, который должен иметь не пустое наполнение. Возможны следующие альтернативы:
1. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@nullFlavor имеет значение "NAV", ТО атрибут identity:Number/@nullFlavor должен иметь значение "NAV";
2. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/ associatedEntity[@classCode="GUAR"]/identity:DocInfo/identity:Insurance PolicyType/@nullFlavor имеет значение, не равное "NAV", ТО элемент identity:Number должен иметь не пустое наполнение;
5.ДОЛЖЕН иметь один [1..1] элемент identity:INN, содержащий ИНН организации или физического лица, который должен иметь не пустое наполнение;
6. ДОЛЖЕН иметь один [1..1] элемент identity:effectiveTime, содержащий срок действия документа-основания. ЕСЛИ атрибут ClinicalDocument/ participant[@typeCode="IND"]/associatedEntity[@classCode="GUAR"]/identit y:DocInfo/identity:InsurancePolicyType/@nullFlavor имеет значение "NAV", ТО атрибут identity:effectiveTime/@nullFlavor должен иметь значение "NAV"; ЕСЛИ кодируется электронный полис ОМС, ТО атрибут identity:effectiveTime/@nullFlavor должен иметь значение "NASK". Данный элемент:
1. ОБЯЗАН иметь один [1..1] элемент identity:low (атрибут @value должен быть представлен с не пустым значением), содержащий дату начала действия документа-основания;
2.ДОЛЖЕН иметь один [1..1] элемент identity:high (атрибут @value должен быть представлен с не пустым значением), содержащий дату окончания действия документа-основания. Возможны следующие альтернативы:
1. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/associatedEntity[@clas sCode="GUAR"]/identity:DocInfo/identity:InsurancePolicyType/@code имеет значение "2", ТО атрибут identity:high/@nullFlavor должен иметь значение "NAV";
2. ЕСЛИ атрибут ClinicalDocument/participant[@typeCode="IND"]/associatedEntity[@clas sCode="GUAR"]/identity:DocInfo/identity:InsurancePolicyType/@code имеет значение "1" или "4", ТО атрибут identity:high/@nullFlavor может быть представлен;

**Reply by eugenekogan-spb** [reply]
Документ представляется атрибутом coverage.identifier
Для полиса ОМС необходимо различать тип полиса (старый, единый бессрочный, единый со сроком, временное свидетельство), поскольку для них разные пространства номеров.
На встрече 15.10.24 решили, что будем отмечать разные типы полисов разными пространствами.

Для полиса ОМС единого бессрочного будет так:
```
* identifier[+]
  * system = "http://fhir.ru/core/systems/OMS_main_unlimited"
  * value = "9876543211234567"
  * type = Core_Cs_Nsi_Coverage_Document#1
```
для временного свидетельства
```
* identifier[+]
  * system = "http://fhir.ru/core/systems/OMS_temp"
  * value = "9876543211234567"
  * type = Core_Cs_Nsi_Coverage_Document#1
```

Кстати, на обсуждение: плохо, что тип не показывает явно, какого типа идентификатор. Может быть, ValueSet для identifier собрать из двух справочников: коды по типам полисов (кроме ОМС) и коды по типам полиса ОМС (для ОМС)?

В этом случае будет (для бессрочного полиса)

```
* identifier[+]
  * system = "http://fhir.ru/core/systems/OMS_main_unlimited"
  * value = "9876543211234567"
  * type = Core_Cs_Nsi_Coverage_Document_OMS#2

CodeSystem: Core_Cs_Nsi_Coverage_Document
* #1 "Полис ОМС"
* #2 "Полис ДМС"

CodeSystem: Core_Cs_Nsi_Coverage_Document_OMS
* #1 "Полис ОМС старого образца"
* #2 "Полис ОМС единый, бессрочный"
* #3 "Полис ОМС единый, со сроком"
* #4 "Временное свидетельство ОМС"

```
@Vrodvi

**Reply by Vrodvi** [reply]
Если следовать рекомендация из FHIR:
![image](https://github.com/user-attachments/assets/e6172746-d8d4-464e-af35-88d007076dc6)
, то получается, что само пространство `OMS_main_unlimited` - уже однозначно определяет тип идентификатора, и в этом случае указание типа не просто дублирует информацию, но и прямо не рекомендуется спецификацией.

По предложенному варианту со справочником нужно удалять пространства идентификаторов и оставлять только значение идентификатор и его тип. Это скорее всего даже будет ближе к действительности, т.к. фактически никакого пространства именно "идентификаторов полисов ОМС единого образца, бессрочных" не существует.

**Reply by eugenekogan-spb** [reply]
Пространства "идентификаторов полисов ОМС единого образца" в открытом доступе нет, но внутри ФФОМС видимо есть, потому что какая-то информационная система формирует эти номера.
В случае, если мы действительно удалим это не известное нам пространство и оставим только тип, не нарушим ли мы правило "идентификатор должен быть уникален по паре system+value"?

**Reply by eugenekogan-spb** [reply]
На встрече 24.12.24 решили: поскольку требование уникальности SHALL, то его нарушать не будем. В будущем, когда/если наши описания пространств идентификаторов станут использоваться, сможем отказаться от type и оставить указание типа по справочнику НСИ только в описании пространства идентификаторов.

**Comment by eugenekogan-spb** [comment]
3.7.3. Сведения об организации

3.МОЖЕТ иметь один [0..1] элемент scopingOrganization, содержащий сведения об организации (страховой компании или юридическом лице), который:
1. ДОЛЖЕН иметь один [1..1] элемент id (атрибут @root должен иметь значение "1.2.643.5.1.13.13.99.2.183", атрибут @extension должен быть представлен с не пустым значением), содержащий уникальный идентификатор страховой компании или юридического лица. Возможны следующие альтернативы:
1. ЕСЛИ кодируется страховая организация, ТО атрибут id/@root должен иметь значение "1.2.643.5.1.13.13.99.2.183", атрибут id/@extension должен иметь значение из справочника НСИ "Реестр страховых медицинских организаций (ФОМС)" (идентификатор справочника "1.2.643.5.1.13.13.99.2.183", поле "Код СМО в едином реестре ОМС");
2. ЕСЛИ кодируется юридическая организация, ТО атрибут id/@nullFlavor должен иметь значение "NA";
2. ОБЯЗАН иметь один [1..1] элемент name, содержащий наименование страховой компании или юридического лица, который должен иметь не пустое наполнение;
3. ДОЛЖЕН иметь хотя бы один [1..*] элемент telecom (атрибут @use может быть представлен с не пустым значением, атрибут @value должен быть представлен с не пустым значением), содержащий контакты страховой компании или юридического лица;
4. ДОЛЖЕН иметь один [1..1] элемент addr (атрибут @use может быть представлен с не пустым значением), содержащий адрес страховой компании или юридического лица, который:
1.ОБЯЗАН иметь один [1..1] элемент streetAddressLine, содержащий адрес (текстом), который должен иметь не пустое наполнение;
2.ОБЯЗАН иметь один [1..1] элемент address:stateCode (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.206", атрибут @codeSystemName должен иметь значение "Субъекты Российской Федерации", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий код субъекта РФ;
3. ДОЛЖЕН иметь один [1..1] элемент postalCode, содержащий почтовый индекс, который должен иметь не пустое наполнение;
4.ДОЛЖЕН иметь один [1..1] элемент fias:Address, содержащий идентификатор адреса по ФИАС, который:
1.ОБЯЗАН иметь один [1..1] элемент fias:AOGUID, содержащий глобальный уникальный идентификатор адресного объекта, который должен иметь не пустое наполнение;
2. ДОЛЖЕН иметь один [1..1] элемент fias:HOUSEGUID, содержащий глобальный уникальный идентификатор дома, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
На встрече 15.10.24 обсудили, что самым правильным было бы указать в качестве идентификатора организации ее код по реестру страховщиков ФФОМС и не приводить ее экземпляр. Но поскольку тип данных reference, а не CodeableReference, то так не сделать, поэтому решили оставить экземпляр.

**Reply by eugenekogan-spb** [reply]
Тем не менее, сам Reference дает возможность указать идентификатор.
Получается: 
```
* payor
  * identifier[+]
    * system = "http://fhir.ru/core/systems/ffoms_smo"
    * value = "22001"
```
Проверку проходит. @Vrodvi Виталий, годится такой вариант?

**Reply by Vrodvi** [reply]
Ага, те самые "логические ссылки", про которые с Олегом недавно обсуждали.

В данном случае, как-будто, их испльзование более уместно - прям полноценного ресурса для СМО нет, но точно есть её "бизнес-идентификатор" (код по ФОМСовскому справочнику). По указанному бизнес-идентификатору, при желании, можно сформировать полноценный ресурс, как того рекондует FHIR - `There is no requirement that a Reference.identifier point to something that is actually exposed as a FHIR instance, but it SHALL point to a business concept that would be expected to be exposed as a FHIR instance, and that instance would need to be of a FHIR resource type allowed by the reference.`, `Reference is intended to point to a structure that can potentially be expressed as a FHIR resource, though there is no need for it to exist as an actual FHIR resource instance`.

![image](https://github.com/user-attachments/assets/f9397ef9-540e-4edb-9141-a730aa835aff)

**Reply by eugenekogan-spb** [reply]
Для r5
```
* paymentBy
  * party
    * identifier[+]
      * system = "http://fhir.ru/core/systems/ffoms_smo"
      * value = "22001"
    * display = "АЛТАЙСКИЙ ФИЛИАЛ ОБЩЕСТВА С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ - СТРАХОВАЯ МЕДИЦИНСКАЯ КОМПАНИЯ РЕСО-МЕД"
```
